tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+minversion = 4.20.0
+envlist = lint,tests,typing
+isolated_build = true
+
+[testenv]
+skip_install = true
+
+[testenv:lint]
+commands=
+  flake8 .
+  black --check --diff .
+  check-manifest .
+  isort --check --diff .
```

Comment:
Do we need the `.`s?

Reply:
it seems like we need it for `black` and `isort`, maybe we can define the SRC by adding some specific config files.

``` console
$ black --check 
Usage: black [OPTIONS] SRC ...

One of 'SRC' or 'code' is required.
```

Reply:
Indeed; let's keep those then.